src/adapters/postgres/cohortMembers-adapter.ts

```diff
@@ -287,48 +285,51 @@ export class PostgresCohortMembersService {
   public async createCohortMembers(
     loginUser: any,
     cohortMembers: CohortMembersDto,
-    response: any
+    res: Response,
+    tenantId: string
   ) {
-
+    const apiId = 'api.post.createCohortMembers';
     try {
       cohortMembers.createdBy = loginUser;
       cohortMembers.updatedBy = loginUser;
 
-      await this.validateEntity(this.cohortRepository, { cohortId: cohortMembers.cohortId }, `Cohort Id does not exist.`);
-      await this.validateEntity(this.usersRepository, { userId: cohortMembers.userId }, `User Id does not exist.`);
-      
+      const existCohort = await this.cohortRepository.find({
+        where: {
+          cohortId: cohortMembers.cohortId
+        }
+      })
+      if (existCohort.length == 0) {
+        return APIResponse.error(res, apiId, "Bad Request", "Invalid input: Cohort Id does not exist.", HttpStatus.BAD_REQUEST;
+      }
+
+      const existUser = await this.usersRepository.find({
+        where: {
+          userId: cohortMembers.userId,
+        }
+      })
+      if (existUser.length == 0) {
+        return APIResponse.error(res, apiId, "Bad Request", "Invalid input: User Id does not exist.", HttpStatus.BAD_REQUEST;
+      }
+
       const existrole = await this.cohortMembersRepository.find({
-        where:{
+        where: {
           userId: cohortMembers.userId,
           cohortId: cohortMembers.cohortId
         }
       })
-      if(existrole.length>0){
-        throw new ErrorResponseTypeOrm({
-          statusCode: HttpStatus.CONFLICT,
-          errorMessage: `This user '${cohortMembers.userId}' already assign for this cohort '${cohortMembers.cohortId}'.`,
-        });
+      if (existrole.length > 0) {
+        return APIResponse.error(res, apiId, "CONFLICT", `User '${cohortMembers.userId}' is already assigned to cohort '${cohortMembers.cohortId}'.`, HttpStatus.CONFLICT;
       }
 
       // Create a new CohortMembers entity and populate it with cohortMembers data
       const savedCohortMember = await this.cohortMembersRepository.save(
         cohortMembers
       );
 
-      return new SuccessResponse({
-        statusCode: HttpStatus.OK,
-        message: "Cohort Member created successfully.",
-        data: savedCohortMember,
-      });
+      return APIResponse.success(res, apiId, savedCohortMember, HttpStatus.OK, "Cohort member has been successfully assigned.");
+
     } catch (e) {
-      if (e instanceof ErrorResponseTypeOrm) {
-        return e;
-      } else {
-        return new ErrorResponseTypeOrm({
-          statusCode: HttpStatus.INTERNAL_SERVER_ERROR,
-          errorMessage: e.toString(), // or any custom error message you want
-        });
-      }
+      return APIResponse.error(res, apiId, "Internal Server Error", `Error is: ${e}`, HttpStatus.INTERNAL_SERVER_ERROR;
```

Comment:
Do not show full error Error is: ${e}`.      
 return APIResponse.error(res, apiId, "Internal Server Error", `${e.errorMsg}`, HttpStatus.INTERNAL_SERVER_ERROR;